### PR TITLE
[Docs] Beam website doc for vertex ai enrichment handler

### PR DESCRIPTION
**Pull request by riteshghorse**
Adds beam webpage for vertex ai enrichment handler and update the transform catalog.

[Staged website](http://apache-beam-website-pull-requests.storage.googleapis.com/30692/documentation/transforms/python/elementwise/enrichment/index.html)

Part of #30386 

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by riteshghorse** [comment]
Made the changes, thanks! Merging now.

**Comment by riteshghorse** [comment]
CC: @rszper

File: website/www/site/content/en/documentation/transforms/python/elementwise/enrichment-bigtable.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <td>
+      <a>
+      {{< button-pydoc path="apache_beam.transforms.enrichment_handlers.bigtable" class="BigTableEnrichmentHandler" >}}
+      </a>
+   </td>
+  </tr>
+</table>
+
+In Apache Beam 2.54.0 and later versions, the enrichment transform includes a built-in enrichment handler for [Bigtable](https://cloud.google.com/bigtable/docs/overview).
+The following example demonstrates how to create a pipeline that use the enrichment transform with [`BigTableEnrichmentHandler`](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.enrichment_handlers.bigtable.html#apache_beam.transforms.enrichment_handlers.bigtable.BigTableEnrichmentHandler).
```
Comment (rszper) [inline website/www/site/content/en/documentation/transforms/python/elementwise/enrichment-bigtable.md:33]:
```suggestion
The following example demonstrates how to create a pipeline that use the enrichment transform with the [`BigTableEnrichmentHandler`](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.enrichment_handlers.bigtable.html#apache_beam.transforms.enrichment_handlers.bigtable.BigTableEnrichmentHandler) handler.
```

File: website/www/site/content/en/documentation/transforms/python/elementwise/enrichment-vertexai.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      <a>
+      {{< button-pydoc path="apache_beam.transforms.enrichment_handlers.vertex_ai_feature_store" class="VertexAIFeatureStoreEnrichmentHandler" >}}
+      </a>
+   </td>
+  </tr>
+</table>
+
+
+In Apache Beam 2.55.0 and later versions, the enrichment transform includes a built-in enrichment handler for [Vertex AI Feature Store](https://cloud.google.com/vertex-ai/docs/featurestore).
+The following example demonstrates how to create a pipeline that use the enrichment transform with [`VertexAIFeatureStoreEnrichmentHandler`](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.enrichment_handlers.vertex_ai_feature_store.html#apache_beam.transforms.enrichment_handlers.vertex_ai_feature_store.VertexAIFeatureStoreEnrichmentHandler) and [`VertexAIFeatureStoreLegacyEnrichmentHandler`](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.enrichment_handlers.vertex_ai_feature_store.html#apache_beam.transforms.enrichment_handlers.vertex_ai_feature_store.VertexAIFeatureStoreLegacyEnrichmentHandler).
```
Comment (rszper) [inline website/www/site/content/en/documentation/transforms/python/elementwise/enrichment-vertexai.md:34]:
```suggestion
The following example demonstrates how to create a pipeline that use the enrichment transform with the [`VertexAIFeatureStoreEnrichmentHandler`](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.enrichment_handlers.vertex_ai_feature_store.html#apache_beam.transforms.enrichment_handlers.vertex_ai_feature_store.VertexAIFeatureStoreEnrichmentHandler) handler and the [`VertexAIFeatureStoreLegacyEnrichmentHandler`](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.enrichment_handlers.vertex_ai_feature_store.html#apache_beam.transforms.enrichment_handlers.vertex_ai_feature_store.VertexAIFeatureStoreLegacyEnrichmentHandler) handler.
```

File: website/www/site/content/en/documentation/transforms/python/elementwise/enrichment-vertexai.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+## Example 1: Enrichment with Vertex AI Feature Store
+
+The precomputed feature values stored in Vertex AI Feature Store uses the following format.
```
Comment (rszper) [inline website/www/site/content/en/documentation/transforms/python/elementwise/enrichment-vertexai.md:38]:
```suggestion
The precomputed feature values stored in Vertex AI Feature Store uses the following format:
```

File: website/www/site/content/en/documentation/transforms/python/elementwise/enrichment-vertexai.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+{{< paragraph class="notebook-skip" >}}
+Output:
+{{< /paragraph >}}
+{{< highlight class="notebook-skip" >}}
+{{< code_sample "sdks/python/apache_beam/examples/snippets/transforms/elementwise/enrichment_test.py" enrichment_with_vertex_ai >}}
+{{< /highlight >}}
+
+## Example 2: Enrichment with Vertex AI Feature Store (Legacy)
+
+The precomputed feature values stored in Vertex AI Feature Store (Legacy) uses the following format:
```
Comment (rszper) [inline website/www/site/content/en/documentation/transforms/python/elementwise/enrichment-vertexai.md:63]:
```suggestion
The precomputed feature values stored in Vertex AI Feature Store (Legacy) use the following format:
```

File: website/www/site/content/en/documentation/transforms/python/elementwise/enrichment-vertexai.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+{{</ highlight >}}
+
+{{< paragraph class="notebook-skip" >}}
+Output:
+{{< /paragraph >}}
+{{< highlight class="notebook-skip" >}}
+{{< code_sample "sdks/python/apache_beam/examples/snippets/transforms/elementwise/enrichment_test.py" enrichment_with_vertex_ai >}}
+{{< /highlight >}}
+
+## Example 2: Enrichment with Vertex AI Feature Store (Legacy)
```
Comment (rszper) [inline website/www/site/content/en/documentation/transforms/python/elementwise/enrichment-vertexai.md:61]:
```suggestion
## Example 2: Enrichment with Vertex AI Feature Store (legacy)
```

File: website/www/site/content/en/documentation/transforms/python/elementwise/enrichment.md [hunk 6]
```diff
@@ -32,35 +32,22 @@ limitations under the License.
...
 
-In Apache Beam 2.54.0 and later versions, the transform includes a built-in enrichment handler for [Bigtable](https://cloud.google.com/bigtable/docs/overview).
+This transform is available in Apache Beam 2.54.0 and later versions.
 
-## Use Bigtable to enrich data
+## Examples
 
-The following example demonstrates how to create a pipeline that use the enrichment transform with [`BigTableEnrichmentHandler`](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.enrichment_handlers.bigtable.html#apache_beam.transforms.enrichment_handlers.bigtable.BigTableEnrichmentHandler).
+The following examples demonstrates how to create a pipeline that use the enrichment transform to enrich data from external services.
```
Comment (rszper) [inline website/www/site/content/en/documentation/transforms/python/elementwise/enrichment.md:39]:
```suggestion
The following examples demonstrate how to create a pipeline that use the enrichment transform to enrich data from external services.
```

**Review by rszper — APPROVED** [review]

**Comment by riteshghorse** [comment]
Thanks for the review! 

Unrelated test failure with timeout. Merging!

**Pull request merged**